src/test/java/com/formstack/qa/tests/AbstractNewUserTest.java

```diff
@@ -0,0 +1,24 @@
+package com.formstack.qa.tests;
+
+import com.formstack.qa.model.SignupData;
+import com.formstack.qa.model.factories.SignupFactory;
+import com.formstack.qa.utils.Utils;
+import org.apache.log4j.Logger;
+import org.testng.annotations.BeforeMethod;
+
+
+public class AbstractNewUserTest extends AbstractTest {
+
+    final static Logger logger = Logger.getLogger(AbstractNewUserTest.class);
```

Comment:
We should be able to have per class loggers if we define it this way:
```suggestion
    protected final Logger logger = Logger.getLogger(getClass());
```

With your solution, we either need to overload logger with every child class, or we'll get logs in the abstract class context, and that's not what we want.

Reply:
So if we do it the way you're suggesting we can remove the logger from every child class?

Reply:
Yes, that's the whole idea behind it. https://stackoverflow.com/a/6653577 has a bit more information.